devicetypes/smartthings/zwave-switch-secure.src/zwave-switch-secure.groovy

```diff
@@ -119,7 +117,7 @@ def refresh() {
 }
 
 private command(physicalgraph.zwave.Command cmd) {
-	if (state.sec != 0) {
+	if (zwaveInfo && zwaveInfo.zw?.endsWith("s")) {
```

Comment:
Some devices using this DTH were added before the new fingerprint type that works with zwaveInfo, so this should be `if ((zwaveInfo.zw == null && state.sec != 0) || zwaveInfo.zw?.endsWIth("s"))`

Reply:
@mckeed 
I modifications requested by you and Donald. I have a question: Do the devices that use this DTH since before the new fingerprint type have zwaveInfo at all? Or should it be something like: 
if ((zwaveInfo == null || zwaveInfo.zw == null && state.sec !=0) || ...
?

Reply:
The implementation of `zwaveInfo` returns an empty map as the failure case, so it should never be null.